db-cleanup/airflow-db-cleanup.py

```diff
@@ -366,10 +370,9 @@ def cleanup_function(**context):
 for db_object in DATABASE_OBJECTS:
```

Comment:
This doesn't work if you aren't using the CeleryExecutor as Task and TaskSet don't exist.  There needs to be a conditional check like on line 247.